### PR TITLE
added staking pool display information and matching to keybase.io

### PR DESCRIPTION
**Pull request by raduchis**
#### Did you test the changes locally ?
- [x] yes
- [ ] no

#### Which category(categories) does this pull request belong to?
- [x] document new feature
- [ ] update documentation that is not relevant anymore
- [x] add examples or more information about a component
- [ ] fix grammar issues
- [ ] other

### CODE REVIEW

File: docs/validators/delegation-manager.md [hunk 1]
```diff
@@ -96,6 +96,39 @@ SetMetadataTransaction {
...
+:::tip
+To be able to connect a **testnet** or **devnet** contract to a keybase.io identity a new folder named `"testnet"` for the testnet or `"devnet"` for the devnet has to be created inside the `"elrond"` folder. An example for the same delegation contract would be `"public/elrondstaking/elrond/testnet/erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqr0llllsj732py"`.
+:::
+
+An example of how the delegation contract will be displayed based on the information provided in the keybase.io is provided below.
+
+![stakingpool](/img/stakingpool.png)
+
+:::important
+To be able to connect also the validators to a specific keybase.io staking pool identity, for every validator an empty file with the name set to the BLS key has to be created and added to the `/elrond` folder. Also the `"identity"` of the validator has to be set to the keybase.io staking pool identity.
```
Comment (SebastianMarian) [inline docs/validators/delegation-manager.md:130]: Also the `"identity"` of the validator has to be set to the prefs.toml file ?

Comment (raduchis) [reply docs/validators/delegation-manager.md]: added info about the prefs.toml

**Review by SebastianMarian — APPROVED** [review]

File: docs/validators/delegation-manager.md [hunk 2]
```diff
@@ -96,6 +96,47 @@ SetMetadataTransaction {
...
+    "@656c726f6e647374616b696e67"       //elrondstaking
+```
+
+:::important
+Setting the keybase.io identity of the staking pool in the metadata is the first step in connecting the delegation contract to a keybase.io identity. The second step is explained in the next chapter [display information](/validators/delegation-manager#display-information) where the inverse connection is made: from the keybase.io identity to the delegation contract address. 
+:::
+
+### Display Information
+
+To customize the information for your delegation contract that will be available in the lists displayed on the delegation pages both in Maiar and the webwallet some aditional information has to be added on the keybase.io account. Please fill in the **avatar picture** and edit the profile providing the **name** and **bio**. This information together with the **service fee, percentage filled** and **APR** will be displayed for every delegation contracton the delegation pages in the webwallet and Maiar. If these information cannot be found a generic logo and the delegation contract's contract address is displayed.
```
Comment (SeverMM) [inline docs/validators/delegation-manager.md:113]: To customize the information for your delegation contract that will be available in the lists displayed on the delegation pages both in Maiar and the webwallet some aditional information has to be added on the keybase.io account.

add 2 commas:

To customize the information for your delegation contract, which will be available in the lists displayed on the delegation pages both in Maiar and the webwallet, some aditional information has to be added on the keybase.io account.

File: docs/validators/delegation-manager.md [hunk 3]
```diff
@@ -96,6 +96,47 @@ SetMetadataTransaction {
...
+
+In order to complete the matching between the delegation contract and keybase.io identity of the staking pool an empty file with the name set to the delegation contract's contract address has to be added in the `/public/<keybase.io identity>/elrond/` folder. An example for the path to the empty file for the `"elrondstaking"` keybase.io identity would be `public/elrondstaking/elrond/erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqr0llllsj732py`.
+
+:::important
+The second and final step in connecting the delegation contract and the keybase.io identity is finalized by adding the empty file that has the name equal to the delegation contract's address and certifying that the referenced keybase.io identity set in the [metadata](/validators/delegation-manager#metadata) also references the delegation contract address. This way the connection from both directions is sealed.
```
Comment (SeverMM) [inline docs/validators/delegation-manager.md:118]: The second and final step in connecting the delegation contract and the keybase.io identity is finalized by adding the empty file that has the name equal to the delegation contract's address and certifying that the referenced keybase.io identity set in the [metadata](/validators/delegation-manager#metadata) also references the delegation contract address.

add some commas:

The second and final step in connecting the delegation contract and the keybase.io identity is finalized by adding the empty file, which has the name equal to the delegation contract's address, and certifying that the referenced keybase.io identity, set in the [metadata](/validators/delegation-manager#metadata), also references the delegation contract address.

File: docs/validators/delegation-manager.md [hunk 4]
```diff
@@ -96,6 +96,47 @@ SetMetadataTransaction {
...
+:::
+
+:::tip
+To be able to connect a **testnet** or **devnet** contract to a keybase.io identity a new folder named `"testnet"` for the testnet or `"devnet"` for the devnet has to be created inside the `/elrond` folder. An example for the same delegation contract would be `public/elrondstaking/elrond/testnet/erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqr0llllsj732py`.
```
Comment (SeverMM) [inline docs/validators/delegation-manager.md:122]: To be able to connect a **testnet** or **devnet** contract to a keybase.io identity a new folder named `"testnet"` for the testnet or `"devnet"` for the devnet has to be created inside the `/elrond` folder.

add commas:

To be able to connect a **testnet** or **devnet** contract to a keybase.io identity, a new folder - named `"testnet"` for the testnet, or `"devnet"` for the devnet - has to be created inside the `/elrond` folder.

**Pull request merged**